### ISSUE TITLE
missing form input labels

### ISSUE THREAD

**Issue by susanev** [chef/automate#3818]
## overview
fix some a11y issues across automate related to inputs missing labels. these were found using the [wave chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US).

## details
- [ ] compliance reports filter search needs a hidden label , there are two different inputs there that need labels
- [ ] compliance copy url input needs label
- [ ] license lockout first and last name have input label issues
- [ ] compliance profile search needs a hidden label of `Search profiles`
- [ ] data lifecycle needs hidden labels for all inputs
- [ ] attributes needs hidden label of `Search attributes`
